wgpu-core/Cargo.toml

```diff
@@ -40,6 +40,9 @@ serde = { version = "1.0", features = ["serde_derive"], optional = true }
 smallvec = "1"
 vec_map = "0.8"
 
+[target.'cfg(loom)'.dependencies]
+loom = "0.3"
```

Comment:
I wonder why this isn't just an optional dependency?

Reply:
I honestly don't really know 😂, I was just using the suggested "getting started" setup from the loom README. Maybe the idea is that, since you already need to have a loom cfg, you might as well use it to gate the dependency? I don't mind changing it. You're saying it could be an optional dependency without being gated on cfg(loom)?

Reply:
Yes, unless there is a specific reason, it should be just an optional dependency that's checked with `cfg(feature = "loom")`

Reply:
Okay, I think I set it up the right way now. Another possibility, maybe for the future: if we want to always run loom tests, we could just have it as a dev dependency and use `#[cfg(test)]` instead of `#[cfg(feature = "loom")]`.

Reply:
Actually, that's a good way to proceed. Let's just have it as a dev dependency, non-conditionally.